src/UserAdmin/src/Smart.FA.Catalog.AccountSimulation/AccountUser.cs

```diff
@@ -0,0 +1,14 @@
+using Microsoft.AspNetCore.Mvc.Rendering;
+
+namespace Smart.FA.Catalog.AccountSimulation;
+
+public static class AccountUsers
+{
+    public static List<SelectListItem> List => new()
```

Comment:
Better to indicate that this prop returns list items and not entities
```suggestion
    public static List<SelectListItem> SelectListItems => new()
```

Reply:
Hm this could have just been in the IndexPageModel but either way, fine for me.